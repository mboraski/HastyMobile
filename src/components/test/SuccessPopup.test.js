// import React from 'react';
// import 'react-native';
// import { shallow } from 'enzyme';
// import SuccessPopup from '../SuccessPopup';
//
// describe('SuccessPopup', () => {
//     it('renders correctly', () => {
//         const render = shallow(<SuccessPopup openModal closeModal={() => {}} />);
//         expect(render).toMatchSnapshot();
//     });
// });
