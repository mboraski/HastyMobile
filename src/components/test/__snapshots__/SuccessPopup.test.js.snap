// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessPopup renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Component
    animationType="slide"
    hardwareAccelerated={false}
    onRequestClose={[Function]}
    style={undefined}
    transparent={true}
    visible={false}
>
    <View
        style={
            Object {
                "alignItems": "center",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "flex": 1,
                "justifyContent": "center",
              }
        }
    >
        <TouchableOpacity
            activeOpacity={1}
            focusedOpacity={0.7}
            onPress={[Function]}
            style={
                Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  }
            }
        >
            <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                    Object {
                        "color": "rgba(0, 0, 0, 0)",
                      }
                }
            >
                .
            </Text>
        </TouchableOpacity>
        <View
            style={
                Object {
                    "alignSelf": "stretch",
                    "backgroundColor": "#ffffff",
                    "borderRadius": 4,
                    "justifyContent": "center",
                    "marginHorizontal": 22,
                    "paddingHorizontal": 17,
                    "paddingVertical": 18.08,
                    "shadowColor": "#000",
                    "shadowOffset": Object {
                      "height": 10,
                      "width": 0,
                    },
                    "shadowOpacity": 0.5,
                    "shadowRadius": 24,
                  }
            }
        >
            <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                    Object {
                        "fontSize": 17.28,
                        "marginVertical": 25.28,
                        "textAlign": "center",
                      }
                }
            >
                Successful
            </Text>
            <Image
                resizeMode="contain"
                source={1}
                style={
                    Object {
                        "alignSelf": "center",
                        "height": 55.04,
                        "width": 55.04,
                      }
                }
            />
            <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                    Object {
                        "fontSize": 17.28,
                        "marginVertical": 25.28,
                        "textAlign": "center",
                      }
                }
            >
                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
            </Text>
            <TouchableOpacity
                activeOpacity={0.2}
                focusedOpacity={0.7}
                onPress={[Function]}
                style={
                    Array [
                        Object {
                          "alignItems": "center",
                          "borderRadius": 20,
                          "height": 40,
                          "justifyContent": "center",
                          "marginHorizontal": 33,
                          "marginVertical": 12,
                        },
                        Object {
                          "backgroundColor": "#000000",
                        },
                      ]
                }
            >
                <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                        Array [
                            Object {
                              "fontSize": 15.36,
                            },
                            Object {
                              "color": "#ffffff",
                            },
                          ]
                    }
                >
                    Apply
                </Text>
            </TouchableOpacity>
            <TouchableOpacity
                activeOpacity={0.2}
                focusedOpacity={0.7}
                onPress={[Function]}
                style={
                    Array [
                        Object {
                          "alignItems": "center",
                          "borderRadius": 20,
                          "height": 40,
                          "justifyContent": "center",
                          "marginHorizontal": 33,
                          "marginVertical": 12,
                        },
                        Object {
                          "backgroundColor": "rgba(0, 0, 0, 0)",
                        },
                      ]
                }
            >
                <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                        Array [
                            Object {
                              "fontSize": 15.36,
                            },
                            Object {
                              "color": "#000000",
                            },
                          ]
                    }
                >
                    Cancel
                </Text>
            </TouchableOpacity>
        </View>
    </View>
</Component>,
  "nodes": Array [
    <Component
      animationType="slide"
      hardwareAccelerated={false}
      onRequestClose={[Function]}
      style={undefined}
      transparent={true}
      visible={false}
>
      <View
            style={
                  Object {
                        "alignItems": "center",
                        "backgroundColor": "rgba(0, 0, 0, 0)",
                        "flex": 1,
                        "justifyContent": "center",
                      }
            }
      >
            <TouchableOpacity
                  activeOpacity={1}
                  focusedOpacity={0.7}
                  onPress={[Function]}
                  style={
                        Object {
                              "bottom": 0,
                              "left": 0,
                              "position": "absolute",
                              "right": 0,
                              "top": 0,
                            }
                  }
            >
                  <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                              Object {
                                    "color": "rgba(0, 0, 0, 0)",
                                  }
                        }
                  >
                        .
                  </Text>
            </TouchableOpacity>
            <View
                  style={
                        Object {
                              "alignSelf": "stretch",
                              "backgroundColor": "#ffffff",
                              "borderRadius": 4,
                              "justifyContent": "center",
                              "marginHorizontal": 22,
                              "paddingHorizontal": 17,
                              "paddingVertical": 18.08,
                              "shadowColor": "#000",
                              "shadowOffset": Object {
                                "height": 10,
                                "width": 0,
                              },
                              "shadowOpacity": 0.5,
                              "shadowRadius": 24,
                            }
                  }
            >
                  <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                              Object {
                                    "fontSize": 17.28,
                                    "marginVertical": 25.28,
                                    "textAlign": "center",
                                  }
                        }
                  >
                        Successful
                  </Text>
                  <Image
                        resizeMode="contain"
                        source={1}
                        style={
                              Object {
                                    "alignSelf": "center",
                                    "height": 55.04,
                                    "width": 55.04,
                                  }
                        }
                  />
                  <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                              Object {
                                    "fontSize": 17.28,
                                    "marginVertical": 25.28,
                                    "textAlign": "center",
                                  }
                        }
                  >
                        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                  </Text>
                  <TouchableOpacity
                        activeOpacity={0.2}
                        focusedOpacity={0.7}
                        onPress={[Function]}
                        style={
                              Array [
                                    Object {
                                      "alignItems": "center",
                                      "borderRadius": 20,
                                      "height": 40,
                                      "justifyContent": "center",
                                      "marginHorizontal": 33,
                                      "marginVertical": 12,
                                    },
                                    Object {
                                      "backgroundColor": "#000000",
                                    },
                                  ]
                        }
                  >
                        <Text
                              accessible={true}
                              allowFontScaling={true}
                              ellipsizeMode="tail"
                              style={
                                    Array [
                                          Object {
                                            "fontSize": 15.36,
                                          },
                                          Object {
                                            "color": "#ffffff",
                                          },
                                        ]
                              }
                        >
                              Apply
                        </Text>
                  </TouchableOpacity>
                  <TouchableOpacity
                        activeOpacity={0.2}
                        focusedOpacity={0.7}
                        onPress={[Function]}
                        style={
                              Array [
                                    Object {
                                      "alignItems": "center",
                                      "borderRadius": 20,
                                      "height": 40,
                                      "justifyContent": "center",
                                      "marginHorizontal": 33,
                                      "marginVertical": 12,
                                    },
                                    Object {
                                      "backgroundColor": "rgba(0, 0, 0, 0)",
                                    },
                                  ]
                        }
                  >
                        <Text
                              accessible={true}
                              allowFontScaling={true}
                              ellipsizeMode="tail"
                              style={
                                    Array [
                                          Object {
                                            "fontSize": 15.36,
                                          },
                                          Object {
                                            "color": "#000000",
                                          },
                                        ]
                              }
                        >
                              Cancel
                        </Text>
                  </TouchableOpacity>
            </View>
      </View>
</Component>,
  ],
  "options": Object {
    "context": Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
    },
  },
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
      },
      "_currentElement": <SuccessPopup
        closeModal={[Function]}
        openModal={true}
/>,
      "_debugID": 3,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": SuccessPopup {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "closeModal": [Function],
          "openModal": true,
        },
        "refs": Object {},
        "state": Object {
          "modalVisible": false,
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 2,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Component
          animationType="slide"
          hardwareAccelerated={false}
          onRequestClose={[Function]}
          style={undefined}
          transparent={true}
          visible={false}
>
          <View
                    style={
                              Object {
                                        "alignItems": "center",
                                        "backgroundColor": "rgba(0, 0, 0, 0)",
                                        "flex": 1,
                                        "justifyContent": "center",
                                      }
                    }
          >
                    <TouchableOpacity
                              activeOpacity={1}
                              focusedOpacity={0.7}
                              onPress={[Function]}
                              style={
                                        Object {
                                                  "bottom": 0,
                                                  "left": 0,
                                                  "position": "absolute",
                                                  "right": 0,
                                                  "top": 0,
                                                }
                              }
                    >
                              <Text
                                        accessible={true}
                                        allowFontScaling={true}
                                        ellipsizeMode="tail"
                                        style={
                                                  Object {
                                                            "color": "rgba(0, 0, 0, 0)",
                                                          }
                                        }
                              >
                                        .
                              </Text>
                    </TouchableOpacity>
                    <View
                              style={
                                        Object {
                                                  "alignSelf": "stretch",
                                                  "backgroundColor": "#ffffff",
                                                  "borderRadius": 4,
                                                  "justifyContent": "center",
                                                  "marginHorizontal": 22,
                                                  "paddingHorizontal": 17,
                                                  "paddingVertical": 18.08,
                                                  "shadowColor": "#000",
                                                  "shadowOffset": Object {
                                                    "height": 10,
                                                    "width": 0,
                                                  },
                                                  "shadowOpacity": 0.5,
                                                  "shadowRadius": 24,
                                                }
                              }
                    >
                              <Text
                                        accessible={true}
                                        allowFontScaling={true}
                                        ellipsizeMode="tail"
                                        style={
                                                  Object {
                                                            "fontSize": 17.28,
                                                            "marginVertical": 25.28,
                                                            "textAlign": "center",
                                                          }
                                        }
                              >
                                        Successful
                              </Text>
                              <Image
                                        resizeMode="contain"
                                        source={1}
                                        style={
                                                  Object {
                                                            "alignSelf": "center",
                                                            "height": 55.04,
                                                            "width": 55.04,
                                                          }
                                        }
                              />
                              <Text
                                        accessible={true}
                                        allowFontScaling={true}
                                        ellipsizeMode="tail"
                                        style={
                                                  Object {
                                                            "fontSize": 17.28,
                                                            "marginVertical": 25.28,
                                                            "textAlign": "center",
                                                          }
                                        }
                              >
                                        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                              </Text>
                              <TouchableOpacity
                                        activeOpacity={0.2}
                                        focusedOpacity={0.7}
                                        onPress={[Function]}
                                        style={
                                                  Array [
                                                            Object {
                                                              "alignItems": "center",
                                                              "borderRadius": 20,
                                                              "height": 40,
                                                              "justifyContent": "center",
                                                              "marginHorizontal": 33,
                                                              "marginVertical": 12,
                                                            },
                                                            Object {
                                                              "backgroundColor": "#000000",
                                                            },
                                                          ]
                                        }
                              >
                                        <Text
                                                  accessible={true}
                                                  allowFontScaling={true}
                                                  ellipsizeMode="tail"
                                                  style={
                                                            Array [
                                                                      Object {
                                                                        "fontSize": 15.36,
                                                                      },
                                                                      Object {
                                                                        "color": "#ffffff",
                                                                      },
                                                                    ]
                                                  }
                                        >
                                                  Apply
                                        </Text>
                              </TouchableOpacity>
                              <TouchableOpacity
                                        activeOpacity={0.2}
                                        focusedOpacity={0.7}
                                        onPress={[Function]}
                                        style={
                                                  Array [
                                                            Object {
                                                              "alignItems": "center",
                                                              "borderRadius": 20,
                                                              "height": 40,
                                                              "justifyContent": "center",
                                                              "marginHorizontal": 33,
                                                              "marginVertical": 12,
                                                            },
                                                            Object {
                                                              "backgroundColor": "rgba(0, 0, 0, 0)",
                                                            },
                                                          ]
                                        }
                              >
                                        <Text
                                                  accessible={true}
                                                  allowFontScaling={true}
                                                  ellipsizeMode="tail"
                                                  style={
                                                            Array [
                                                                      Object {
                                                                        "fontSize": 15.36,
                                                                      },
                                                                      Object {
                                                                        "color": "#000000",
                                                                      },
                                                                    ]
                                                  }
                                        >
                                                  Cancel
                                        </Text>
                              </TouchableOpacity>
                    </View>
          </View>
</Component>,
        "_debugID": 4,
        "_renderedOutput": <Component
          animationType="slide"
          hardwareAccelerated={false}
          onRequestClose={[Function]}
          style={undefined}
          transparent={true}
          visible={false}
>
          <View
                    style={
                              Object {
                                        "alignItems": "center",
                                        "backgroundColor": "rgba(0, 0, 0, 0)",
                                        "flex": 1,
                                        "justifyContent": "center",
                                      }
                    }
          >
                    <TouchableOpacity
                              activeOpacity={1}
                              focusedOpacity={0.7}
                              onPress={[Function]}
                              style={
                                        Object {
                                                  "bottom": 0,
                                                  "left": 0,
                                                  "position": "absolute",
                                                  "right": 0,
                                                  "top": 0,
                                                }
                              }
                    >
                              <Text
                                        accessible={true}
                                        allowFontScaling={true}
                                        ellipsizeMode="tail"
                                        style={
                                                  Object {
                                                            "color": "rgba(0, 0, 0, 0)",
                                                          }
                                        }
                              >
                                        .
                              </Text>
                    </TouchableOpacity>
                    <View
                              style={
                                        Object {
                                                  "alignSelf": "stretch",
                                                  "backgroundColor": "#ffffff",
                                                  "borderRadius": 4,
                                                  "justifyContent": "center",
                                                  "marginHorizontal": 22,
                                                  "paddingHorizontal": 17,
                                                  "paddingVertical": 18.08,
                                                  "shadowColor": "#000",
                                                  "shadowOffset": Object {
                                                    "height": 10,
                                                    "width": 0,
                                                  },
                                                  "shadowOpacity": 0.5,
                                                  "shadowRadius": 24,
                                                }
                              }
                    >
                              <Text
                                        accessible={true}
                                        allowFontScaling={true}
                                        ellipsizeMode="tail"
                                        style={
                                                  Object {
                                                            "fontSize": 17.28,
                                                            "marginVertical": 25.28,
                                                            "textAlign": "center",
                                                          }
                                        }
                              >
                                        Successful
                              </Text>
                              <Image
                                        resizeMode="contain"
                                        source={1}
                                        style={
                                                  Object {
                                                            "alignSelf": "center",
                                                            "height": 55.04,
                                                            "width": 55.04,
                                                          }
                                        }
                              />
                              <Text
                                        accessible={true}
                                        allowFontScaling={true}
                                        ellipsizeMode="tail"
                                        style={
                                                  Object {
                                                            "fontSize": 17.28,
                                                            "marginVertical": 25.28,
                                                            "textAlign": "center",
                                                          }
                                        }
                              >
                                        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                              </Text>
                              <TouchableOpacity
                                        activeOpacity={0.2}
                                        focusedOpacity={0.7}
                                        onPress={[Function]}
                                        style={
                                                  Array [
                                                            Object {
                                                              "alignItems": "center",
                                                              "borderRadius": 20,
                                                              "height": 40,
                                                              "justifyContent": "center",
                                                              "marginHorizontal": 33,
                                                              "marginVertical": 12,
                                                            },
                                                            Object {
                                                              "backgroundColor": "#000000",
                                                            },
                                                          ]
                                        }
                              >
                                        <Text
                                                  accessible={true}
                                                  allowFontScaling={true}
                                                  ellipsizeMode="tail"
                                                  style={
                                                            Array [
                                                                      Object {
                                                                        "fontSize": 15.36,
                                                                      },
                                                                      Object {
                                                                        "color": "#ffffff",
                                                                      },
                                                                    ]
                                                  }
                                        >
                                                  Apply
                                        </Text>
                              </TouchableOpacity>
                              <TouchableOpacity
                                        activeOpacity={0.2}
                                        focusedOpacity={0.7}
                                        onPress={[Function]}
                                        style={
                                                  Array [
                                                            Object {
                                                              "alignItems": "center",
                                                              "borderRadius": 20,
                                                              "height": 40,
                                                              "justifyContent": "center",
                                                              "marginHorizontal": 33,
                                                              "marginVertical": 12,
                                                            },
                                                            Object {
                                                              "backgroundColor": "rgba(0, 0, 0, 0)",
                                                            },
                                                          ]
                                        }
                              >
                                        <Text
                                                  accessible={true}
                                                  allowFontScaling={true}
                                                  ellipsizeMode="tail"
                                                  style={
                                                            Array [
                                                                      Object {
                                                                        "fontSize": 15.36,
                                                                      },
                                                                      Object {
                                                                        "color": "#000000",
                                                                      },
                                                                    ]
                                                  }
                                        >
                                                  Cancel
                                        </Text>
                              </TouchableOpacity>
                    </View>
          </View>
</Component>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <SuccessPopup
    closeModal={[Function]}
    openModal={true}
/>,
}
`;
